### DISCUSSION TITLE
Expansion of the Random Set feature

### DISCUSSION THREAD

**Discussion by BloodParade** [RetroAchievements/RAWeb#1885] [Ideas]
This would probably be a really low priority thing but, when I tried to set up a random set tour manually via Google Sheets, it was a pretty annoying procedure, and keeping the sheets constantly up-to-date with all the new sets just doesn't seem feasible, especially with my skill.

It'd be pretty neat if the Random Set option could potentially get expanded into a full-on randomizer via potential filters, ex. a random set from just a specific console, or only sets you haven't played/mastered yet.

**Comment by Jamiras** [comment]
I believe this is addressed by #2866, where you can build a list of games using filters, then have something randomly selected from that list.

**Comment by wescopeland** [comment]
This is indeed covered by the "Surprise me" button.
